### PR TITLE
Rename tryFinishTrack to canFinishRace for better expectations

### PR DESCRIPTION
**Pull request by gigaSproule**
# pull request

Renamed the `tryFinishTrack` in `NeedForSpeed` to be `canFinishRace` to match the expectation of the instructions for the final task.

Resolves https://github.com/exercism/java-analyzer/issues/184.

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/java/blob/main/POLICIES.md#event-checklist)

### CODE REVIEW

File: exercises/concept/need-for-speed/src/test/java/NeedForSpeedTest.java [hunk 1]
```diff
@@ -195,12 +195,12 @@ public void car_can_finish_with_car_that_just_cannot_finish() {
...
         var race = new RaceTrack(distance);
 
-        assertThat(race.tryFinishTrack(car)).isFalse();
+        assertThat(race.canFinishRace(car)).isFalse();
     }
 
     @Test
     @Tag("task:6")
-    @DisplayName("The tryFinishCar method returns false when car cannot finish a race")
+    @DisplayName("The  method returns false when car cannot finish a race")
```
Comment (kahgoh) [inline exercises/concept/need-for-speed/src/test/java/NeedForSpeedTest.java:203]:
```suggestion
    @DisplayName("The canFinishRace method returns false when car cannot finish a race")
```

Adding the name back makes it consistent with the other tests.

Comment (gigaSproule) [reply exercises/concept/need-for-speed/src/test/java/NeedForSpeedTest.java:203]: Good spot. Not even sure how that got removed, as I replaced everything...

**Comment by manumafe98** [comment]
Yeah we will have to open as well a PR for the analyzer.
Check [this](https://github.com/search?q=repo%3Aexercism%2Fjava-analyzer%20path%3A%2F%5Esrc%5C%2F%2F%20tryFinishTrack&type=code)
If you want to do it as well @gigaSproule go ahead.

**Comment by gigaSproule** [comment]
@manumafe98 I've created https://github.com/exercism/java-analyzer/pull/201.

**Review by kahgoh — APPROVED** [review]

**Pull request merged**